src/components/AreaOfResponsibiltySelector.tsx

```diff
@@ -49,16 +50,19 @@ export const AreaOfResponsibilitySelector = () => {
         return await fetchOrganization(orgId);
       });
       const organizations = await Promise.all(areaPromises);
-      const organizationOptions = buildOrganizationOptions(organizations);
-      if (!searchParams.get(TicketSearchParam.ViewingScope)) {
-        searchParams.set(TicketSearchParam.ViewingScope, organizationOptions.map((org) => org.id).join(','));
-        history.push({ search: searchParams.toString() });
-      }
-      return organizationOptions;
+      return buildOrganizationOptions(organizations);
     },
     meta: { errorMessage: t('feedback.error.get_institution') },
   });
-  const organizationOptions = organizationQuery.data ?? [];
+  const organizationOptions = useMemo(() => organizationQuery.data ?? [], [organizationQuery]);
```

Comment:
god praksis å være mest mulig spesifikk, for å unngå å gjøre det oftere enn nødvendig. I dette tilfelle har det neppe noen praktisk betydning, but still
```suggestion
  const organizationOptions = useMemo(() => organizationQuery.data ?? [], [organizationQuery.data]);
```